### PR TITLE
Removed Bootstrap classes

### LINKED ISSUE #1064: Modal "hidden" class remaining active
Hello, love the library! 

The library works fine for me, with the exception of: using the example modal code (below) my overlay darkens but the `hidden` class remains on the modal html, so it doesn't become visible to the user, could anyone suggest a solution? 

"tailwindcss": "^2.0.3",
"tw-elements": "^1.0.0-alpha12"

```html
<!-- Button trigger modal -->
<button type="button" class="px-6
      py-2.5
      bg-blue-600
      text-white
      font-medium
      text-xs
      leading-tight
      uppercase
      rounded
      shadow-md
      hover:bg-blue-700 hover:shadow-lg
      focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0
      active:bg-blue-800 active:shadow-lg
      transition
      duration-150
      ease-in-out" data-bs-toggle="modal" data-bs-target="#exampleModal">
  Launch demo modal
</button>

<!-- Modal -->
<div class="modal fade fixed top-0 left-0 hidden w-full h-full outline-none overflow-x-hidden overflow-y-auto"
  id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
  <div class="modal-dialog relative w-auto pointer-events-none">
    <div
      class="modal-content border-none shadow-lg relative flex flex-col w-full pointer-events-auto bg-white bg-clip-padding rounded-md outline-none text-current">
      <div
        class="modal-header flex flex-shrink-0 items-center justify-between p-4 border-b border-gray-200 rounded-t-md">
        <h5 class="text-xl font-medium leading-normal text-gray-800" id="exampleModalLabel">Modal title</h5>
        <button type="button"
          class="btn-close box-content w-4 h-4 p-1 text-black border-none rounded-none opacity-50 focus:shadow-none focus:outline-none focus:opacity-100 hover:text-black hover:opacity-75 hover:no-underline"
          data-bs-dismiss="modal" aria-label="Close"></button>
      </div>
      <div class="modal-body relative p-4">
        Modal body text goes here.
      </div>
      <div
        class="modal-footer flex flex-shrink-0 flex-wrap items-center justify-end p-4 border-t border-gray-200 rounded-b-md">
        <button type="button" class="px-6
          py-2.5
          bg-purple-600
          text-white
          font-medium
          text-xs
          leading-tight
          uppercase
          rounded
          shadow-md
          hover:bg-purple-700 hover:shadow-lg
          focus:bg-purple-700 focus:shadow-lg focus:outline-none focus:ring-0
          active:bg-purple-800 active:shadow-lg
          transition
          duration-150
          ease-in-out" data-bs-dismiss="modal">Close</button>
        <button type="button" class="px-6
      py-2.5
      bg-blue-600
      text-white
      font-medium
      text-xs
      leading-tight
      uppercase
      rounded
      shadow-md
      hover:bg-blue-700 hover:shadow-lg
      focus:bg-blue-700 focus:shadow-lg focus:outline-none focus:ring-0
      active:bg-blue-800 active:shadow-lg
      transition
      duration-150
      ease-in-out
      ml-1">Save changes</button>
      </div>
    </div>
  </div>
</div>
```

### LINKED ISSUE #1090: Remove bootstrap and mdb classes from components
In our mdb components (specifically pickers, ripple, stepper, timepicker) we still use bootstrap classes or our custom ones (for example
the classes .form-control, .block, but most of all the dialog classes, i.e. .datepicker-backdrop, .datepicker-modal-container etc). It is similar in bootstrap components, because, for example, in dropdown we use the .dropdown-menu or .dropdown-item classes. All these classes should be rewritten to tw both in html and js (both static and dynamic). When it's ready, we need to remove bsComponents from src / js / plugin.js, because that's where our custom code came in.

Finally, we need to delete the folders ./scss/bs and ./scss/mdb, the index.scss file and also cdn.scss (here we only need to correct the configuration so that we can switch from cdn to taiwind)

**Bootstrap**

- Accordion @juujisai 
- Buttons @Trochonovitz 
- Carousel @r-jacko 
- Close @juujisai 
- Dropdown @juujisai 
- Modal @Trochonovitz 
- Navbar @juujisai 
- Offcanvas @juujisai 
- Popover @Trochonovitz 
- Spinners @r-jacko 
- Tooltip @Trochonovitz 
- Transitions @juujisai 

**MDB**

- Animate @Trochonovitz 
- Collapse @r-jacko 
- Datepicker @juujisai 
- Ripple @juujisai 
- Stepper @Trochonovitz 
- Timepicker @r-jacko

### PR DESCRIPTION
**Pull request by Trochonovitz**

### CODE REVIEW

File: demo/sites/bs/tooltip.html [hunk 1]
```diff
@@ -7,9 +7,9 @@
     <meta name="author" content="Tailwind Elements" />
     <title>Album example · Bootstrap v5.1</title>
 
-    <!-- Roboto font -->
+    <!-- Inter font -->
```
Comment (smolenski-mikolaj) [inline demo/sites/bs/tooltip.html]: It should not change. We want Roboto font finally.

File: src/js/bs/src/util/backdrop.js [hunk 2]
```diff
@@ -56,10 +43,19 @@ class Backdrop {
...
+    this._getElement().classList.add('transition-all');
+    this._getElement().classList.add('duration-300');
+    this._getElement().classList.add('ease-in-out');
+    this._getElement().classList.add('fixed');
+    this._getElement().classList.add('top-0');
+    this._getElement().classList.add('left-0');
+    this._getElement().classList.add('z-[1050]');
+    this._getElement().classList.add('bg-[#000]');
+    this._getElement().classList.add('w-[100vw]');
+    this._getElement().classList.add('h-[100vh]');
```
Comment (smolenski-mikolaj) [inline src/js/bs/src/util/backdrop.js]: There should be a helper function to add multiple classes at once. If there's not, then we should add such a function.

**Review by smolenski-mikolaj — CHANGES_REQUESTED** [review]

File: src/js/mdb/dom/manipulator.js [hunk 3]
```diff
@@ -99,6 +99,12 @@ const Manipulator = {
     element.classList.add(className);
   },
 
+  addManyClasses(element, arrayClasses) {
```
Comment (smolenski-mikolaj) [inline src/js/mdb/dom/manipulator.js]: Please change it to `addMultipleClasses`

**Comment by jdnichollsc** [comment]
@Trochonovitz @smolenski-mikolaj carousel SELECTOR_DATA_ACTIVE_ITEM needs to be updated as well :sweat_smile:

**Comment by Trochonovitz** [comment]
> @Trochonovitz @smolenski-mikolaj carousel SELECTOR_DATA_ACTIVE_ITEM needs to be updated as well 😅

@jdnichollsc could you elaborate?

**Comment by jdnichollsc** [comment]
@Trochonovitz sorry, I was wrong! I was migrating a project and I didn't see that the styles are different now, thanks for your help!

**Pull request merged**